src/openstudio_lib/OSDocument.cpp

```diff
@@ -441,6 +444,7 @@ void OSDocument::createTab(int verticalId) {
   m_verticalId = verticalId;
 
   bool isIP = m_mainWindow->displayIP();
+  bool displayAdditionalProps = m_mainWindow->displayAdditionalProps();
```

Comment:
Does `toggleDisplayAdditionalPropsClicked` get emitted somewhere on startup if it is set?

Reply:
It's read from settings and passed around. I dont think it needs to be emitted